public/js/analytics.js

```diff
@@ -32,15 +43,37 @@
   var str_tx_re = /str_id=(\w+)/;
   var str_amt_re = /str_amount=([\w\.]+)/;
   var str_cc_re = /str_currency=(\w+)/;
+  var str_frequency_re = /str_frequency=(\w+)/;
```

Comment:
We're not actually passing this param through yet for Stripe recurring, but the GA code is ready to go when Stripe is.